lib.sh

```diff
@@ -1048,7 +1048,7 @@ fi
 
 #example: is_app_installed documentserver_community
 is_app_installed() {
-if nextcloud_occ app:list | grep -wq "$1"
+if nextcloud_occ app:list | awk '{print$2}' | tr -d ':' | sed '/^$/d' | grep -q "^$1$"
```

Comment:
why is this better?

Reply:
I mean, now you use 4 tools instead of just one...

Reply:
you are correct. the old method seems to work just fine.
I will revert this